src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -95,22 +96,110 @@ export const InitializeClient: FC<OwnProps> = ({
 
   useEffect(() => {
     const fireKeyboardCopyEvent = event => {
-      if (
-        keyboardCopyTriggered(event) &&
-        (userSelection().includes('influx config create') ||
-          userSelection().includes('influx bucket create'))
-      ) {
+      if (keyboardCopyTriggered(event) && userSelection().includes('#define')) {
         logCopyCodeSnippet()
       }
     }
     document.addEventListener('keydown', fireKeyboardCopyEvent)
     return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
   }, [])
 
+  const codeSnippet = `#if defined(ESP32)
+  #include <WiFiMulti.h>
+  WiFiMulti wifiMulti;
+  #define DEVICE "ESP32"
+  #elif defined(ESP8266)
+  #include <ESP8266WiFiMulti.h>
+  ESP8266WiFiMulti wifiMulti;
+  #define DEVICE "ESP8266"
+  #endif
+  
+  #include <InfluxDbClient.h>
+  #include <InfluxDbCloud.h>
+  
+  // WiFi AP SSID
+  #define WIFI_SSID "YOUR_WIFI_SSID"
+  // WiFi password
+  #define WIFI_PASSWORD "YOUR_WIFI_PASSWORD"
+  
+  #define INFLUXDB_URL "${url}"
+  #define INFLUXDB_TOKEN "${token}"
+  #define INFLUXDB_ORG "${org.id}"
+  #define INFLUXDB_BUCKET "${
+    bucket.name === '<BUCKET>' ? 'YOUR_BUCKET' : bucket.name
+  }"
+  
+  // Time zone info
+  #define TZ_INFO "UTC−07:00"
```

Comment:
should this be hardcoded to -7 UTC (which is Sahas' timezone)? Or should we do something like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset ?

Edit: seems like it, since it says accurate timezone information is necessary.

Reply:
Yes, this needs to be the User's TZ info. 👍

Reply:
Fixed!